backend/tests/users.test.ts

```diff
@@ -380,7 +380,6 @@ describe("Testing /users/:userID/registered", () => {
     );
     const data = GETresponse.body;
     expect(GETresponse.status).toBe(200);
-    expect(data.length).toBe(1);
```

Comment:
To be clear, I removed this line because it is not a clever way to test in this context. We are doing all sorts of things to the seed data so we can't guarantee that the specific user will only be registered to one event